### PR TITLE
Feature/shottable

### PR DESCRIPTION
**Pull request by eliabejr**

### CODE REVIEW

**Comment by eliabejr** [comment]
Screen Shot 2019-03-15 at 11 34 49

**Comment by fakenickels** [comment]
🤔 wouldn't the notification be a JSONTreeViewer?

File: storybook/stories/NotificationStories.re [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Storybook.(
+  storiesOf("Notification", module_)
+  |> add("default", () => <Notification />)
+  |> add("notification share", () =>
+       <View style=Style.(style([flex(1.)]))>
+         <NotificationShare
+           title={j|Notificação 012345|j}
+           eventId="AKDJF19939399291DKD"
+           date={Js.Date.make() |> Js.Date.toString}
+           fields=[|
```
Comment (fakenickels) [inline storybook/stories/NotificationStories.re:14]: we'll be actually the JSON here with the notification stuff right

Comment (fakenickels) [reply storybook/stories/NotificationStories.re:14]: to treeviewer

File: src/Hook.re [hunk 2]
```diff
@@ -0,0 +1,2 @@
+let let_ = (make, children) =>
```
Comment (fakenickels) [inline src/Hook.re]: this is already on the latest version of bs-epitath

File: storybook/stories/NotificationStories.re (hunk 1 above)
Comment (eliabejr) [reply storybook/stories/NotificationStories.re:14]: ahh, top. Thought it would be a "raw" of the notification content

Comment (fakenickels) [reply storybook/stories/NotificationStories.re:14]: yea this way we would need to write a json parser from JSON to the Field variant

Comment (guilhermedecampo) [reply storybook/stories/NotificationStories.re:14]: `What` and `When` could be the way its and the treeviewer already opened could be the `More info`in this structure

**Comment by fakenickels** [comment]
how is it going here @eliabejr ?

File: storybook/stories/NotificationStories.re (hunk 1 above)
Comment (eliabejr) [reply storybook/stories/NotificationStories.re:14]: what you think, guys?
image

**Review by fakenickels — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -15,19 +15,22 @@
     "@storybook/addons": "^4.1.11",
     "@storybook/react-native": "^4.1.11",
     "app-icon": "^0.6.3",
+    "babel-core": "^6.26.3",
+    "babel-runtime": "^6.26.0",
```
Comment (fakenickels) [inline package.json]: https://github.com/Astrocoders/last-card/blob/develop/package.json#L7-L8

**Pull request merged**